pkgs/development/rocm-modules/6/aotriton/default.nix

```diff
@@ -75,19 +75,18 @@ stdenv.mkDerivation (
     env.ROCM_PATH = "${clr}";
     requiredSystemFeatures = [ "big-parallel" ];
 
-    outputs =
```

Comment:
was running nixfmt-rfc-style before I had committed anything a mistake? didn't expect it to change all this :c